test/clj_reload/keep_test.clj

```diff
@@ -129,147 +129,151 @@
     (is (thrown? Exception (tu/reload)))))
 
 (deftest keep-type-test
-  (tu/init 'clj-reload.keep-deftype)
-  (let [ns             (find-ns 'clj-reload.keep-deftype)
-        normal-new     @(ns-resolve ns 'type-normal-new)
-        normal-factory @(ns-resolve ns 'type-normal-factory)
-        keep-new       @(ns-resolve ns 'type-keep-new)
-        keep-factory   @(ns-resolve ns 'type-keep-factory)
-        _              (tu/touch 'clj-reload.keep-deftype)
-        _              (tu/reload)
-        ns'            (find-ns 'clj-reload.keep-deftype)]
-    (is (not= normal-new @(ns-resolve ns' 'type-normal-new)))
-    (is (not (identical? (class normal-new) (class @(ns-resolve ns' 'type-normal-new)))))
-    
-    (is (not= normal-factory @(ns-resolve ns' 'type-normal-factory)))
-    (is (not (identical? (class normal-factory) (class @(ns-resolve ns' 'type-normal-factory)))))
-    
-    (is (not (identical? keep-new @(ns-resolve ns' 'type-keep-new))))
-    (is (= keep-new @(ns-resolve ns' 'type-keep-new)))
-    (is (identical? (class keep-new) (class @(ns-resolve ns' 'type-keep-new))))
+  (when-not (tu/babashka?)
```

Comment:
Compare with github's "Hide whitespace". I'm just not running the test when on bb.